Wox.Plugin.Runner/Services/ExplorerPathsService.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+
+namespace Wox.Plugin.Runner
+{
+    class ExplorerPathsService
+    {
+        public static List<string> GetOpenExplorerPaths()
+        {
+            var results = new List<string>();
+            if (OperatingSystem.IsWindows())
+            {
+                Type? type = Type.GetTypeFromProgID("Shell.Application");
+                if (type == null) return results;
+                dynamic? shell = Activator.CreateInstance(type);
+                if (shell == null) return results;
+                try
+                {
+                    var fullResults = new List<ExplorerResult>();
+
+                    var openWindows = shell.Windows();
+                    for (int i = 0; i < openWindows.Count; i++)
+                    {
+                        var window = openWindows.Item(i);
+                        if (window == null) continue;
+                        var fileName = Path.GetFileName((string)window.FullName);
+                        if (fileName.ToLower() == "explorer.exe")
+                        {
+                            string locationUrl = window.LocationURL;
+                            if (!string.IsNullOrEmpty(locationUrl))
+                            {
+                                fullResults.Add(new ExplorerResult(new IntPtr(window.HWND), new Uri(locationUrl).LocalPath));
+                            }
+                        }
+                    }
+
+                    int zIndex = fullResults.Count;
+                    EnumWindows((IntPtr hwnd, IntPtr param) =>
+                    {
+                        var result = fullResults.Find(v => v.HWND == hwnd);
+                        if(result != null)
+                        {
+                            result.ZIndex = zIndex;
```

Comment:
I wonder why do we want to sort out the windows?
Shouldn't we pick the window with the largest zindex?

Reply:
Oh I messed up with the logic. So the EnumWindow Call will enumerate with the ZIndex order?

Reply:
Do you plan to make usage of the ZIndex? But I think it would be a bit hard to do so lol. Maybe adding multiple results instead of only the first one, but order them based on the zindex? BTW, you can order the results with the `Score` property for `Result` class.

Reply:
I slightly tweaked the code, let me know if it's easier to understand now